server/homebrew.api.js

```diff
@@ -478,14 +493,42 @@ const api = {
 		}
 
 		res.status(204).send();
+	},
+	returnBrew : async (req, res)=>{
+		return res.status(200).json(req.brew);
+	},
+	shareAuthorChecks : async (req, res, next)=>{
+		// If logged in user is not an author:
+		if(!req.brew.authors.includes(req?.account?.username)){
+			// Remove Edit ID
+			req.brew.editId = undefined;
```

Comment:
This appears to be performing part of the `sanatizeBrew()` function (removing the `editId`), but not the other parts (removing `_id` and `__v`). Do we need to handle that here by bringing over `sanatizeBrew()`, or is it already covered somewhere in the fetching process?

Reply:
I've updated the function in the latest commit to also remove the `_id` and `__v` properties.